src/make.ts

```diff
@@ -115,7 +119,8 @@ export function blockedByOp(op: Operations, showPopup: boolean = true): Operatio
     if (getIsConfiguring()) {
         // A configure in the background shouldn't block anything except another configure
         if (getConfigureIsInBackground() && op !== Operations.configure) {
-            vscode.window.showInformationMessage(`The project is configuring in the background and ${op} may run on out-of-date input.`);
+            vscode.window.showInformationMessage(localize("project.configuring.background.op.may.run.on.out.of.date.input",
```

Comment:
>project.configuring.background.op.may.run.on.out.of.date.input [](http://example.com/codeflow?start=59&length=62)

The key names don't need to summarize the message. I would recommend making them shorter.